### ISSUE TITLE
Last synced status not accessible

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#93878]
Refs: https://github.com/microsoft/vscode/issues/93799

Once you are syncing the "Last synced: now" is nicely shown in our Settings UX. However this part of the UX is not accessible, since it can not be focused to and is never read out.
I suggest that we add this to the end of the aria label of the whole settings UX container (since it is global) or that we make it navigable to.

**Comment by roblourens** [comment]
Does it need to be an aria-alert so it reads 'Last Synced: now' every time it changes? I think this would be not that useful but it is missing info.

**Comment by isidorn** [comment]
No, definetley not an aria-alert.
I suggest the other two options: add it to the aria label of the whole container or make the element navigable to.

**Comment by roblourens** [comment]
The container is not focusable with the keyboard so I don't know when that would be read. And I don't want to add a non-functional tab stop for all users. Does it make sense to add it to the aria-label for the search input? Then it would be read when the editor is opened.

**Comment by isidorn** [comment]
@roblourens yes I think it makes sense. Especaily if it it as the end of the search input so users can skip it.
